packages/react-dom/src/__tests__/ReactLegacyErrorBoundaries-test.internal.js

```diff
@@ -0,0 +1,2125 @@
+/**
```

Comment:
This file was literally copied from `ReactErrorBoundaries-test.internal.js` so it doesn't really need to be reviewed. Normally I would have `git mv`ed it, but in this case– I wanted the reviewer to be able to easily spot the places the new render-phase error boundary behavior _differed_ from the old behavior.